examples/html-css/README.md

```diff
@@ -0,0 +1,22 @@
+## Example: Pure HTML/CSS
+
+This shows the usage of CMS design system components on a pure HTML/CSS page with examples of:
+- basic article
+- simple form with submit button
+- using responsive classes 
+
+Simply include the CSS file in the HTML document.
+
+### Usage
+
+```html
+<link rel="stylesheet" href="packages/design-system/dist/css/index.css" />
+```
+
+## Getting started
+
+Install and build the core design system to ensure html-css example can reference the minified CSS.
+
+1. Install packages: `npm install` or `yarn install`
```

Comment:
Same info here, most HTML/CSS apps wont be using the CMSDS via npm, they will be downloading and manually using the CSS file